### PR TITLE
[Dash] Add clear app to admin

### PR DESCRIPTION
**Pull request by nezaj**
[One of our users](https://discord.com/channels/1031957483243188235/1148286343638687786/1288277649805938729) asked if we could make it easy to clear apps. This is especially useful if someone is prototyping on a paid app, since it's not easy for them to re-create an app from scratch.

One thing that was strange was that the explorer would be stale after clearing the data. I get around this by doing a reload, but I'd be open to a better solution, just wasn't sure how.

It might also be nice to offer some more checkboxes / configuration for what things you want to clear, but I figured this is decent solution for now and if folks need something more customizable they will let us know.

I also held off on adding this to the cli-helper, just wanted to make sure this actually meets his needs for now before increasing the surface area of other packages

https://github.com/user-attachments/assets/b2982671-b975-4035-8ccf-e6d7321a06b5

### CODE REVIEW

File: client/www/pages/dash/index.tsx [hunk 1]
```diff
@@ -1288,21 +1290,56 @@ function Admin({
...
+                  await jsonFetch(`${config.apiURI}/dash/apps/${app.id}/clear`, {
+                    method: 'POST',
+                    headers: {
+                      authorization: `Bearer ${token}`,
+                      'content-type': 'application/json',
+                    },
+                  });
+
+                  clearDialog.onClose();
+                  window.location.reload();
```
Comment (dwwoelfel) [inline client/www/pages/dash/index.tsx]: Do we need the reload? If you want to make sure the rules change, you can do:

```
await dashResponse.mutate();
```

Also might be nice to add a success toast here:

```
successToast('App cleared!');
```

**Review by dwwoelfel — APPROVED** [review]
One comment, but LGTM!

File: client/www/pages/dash/index.tsx (hunk 1 above)
Comment (nezaj) [reply client/www/pages/dash/index.tsx]: Thanks @dwwoelfel ! `dashResponse.mutate()` was what I was looking for!

File: server/src/instant/model/app.clj [hunk 2]
```diff
@@ -273,6 +275,17 @@
                             WHERE a.id = ?::uuid"
                            new-creator-id id])))
 
+(defn clear-by-id!
+  "Deletes attrs, rules, and triples for the specified app_id"
+  ([params] (clear-by-id! aurora/conn-pool params))
+  ([conn {:keys [id]}]
+   (next-jdbc/with-transaction [tx-conn conn]
+     (attr-model/delete-by-app-id! tx-conn id)
+     (rule-model/delete-by-app-id! tx-conn {:app-id id}))))
```
Comment (dwwoelfel) [inline server/src/instant/model/app.clj]: To get this to invalidate queries, you probably need to add:

```
(transaction-model/create! conn {:app-id id})
```

inside the transaction.

**Pull request merged**